internal/common/shared/sanctions.go

```diff
@@ -0,0 +1,32 @@
+package shared
+
+import "github.com/SOMAS2020/SOMAS2020/internal/common/rules"
+
+// IIGOSanctionsScore provides typed integer score for each island
+type IIGOSanctionsScore int
+
+// IIGOSanctionsTier provides typed integer tiers for sanctions
+type IIGOSanctionsTier int
+
+// Provides enumerated tiers for IIGO sanctions
+const (
+	SanctionTier1 IIGOSanctionsTier = iota
+	SanctionTier2
+	SanctionTier3
+	SanctionTier4
+	SanctionTier5
+	NoSanction
+)
```

Comment:
ah missed this--can you do the `String` and `Marshal` functions as per #150?

Reply:
(if this is inside gamestate)

Reply:
sure thing! done :)